package.json

```diff
@@ -40,6 +40,7 @@
   },
   "github": "https://github.com/Mikhus/domurl",
   "devDependencies": {
+    "jest": "^28.1.0",
```

Comment:
Why is this necessary? We are using mocha.

Reply:
It is unnecessary, it could be removed.

Reply:
OK